pub-kube-config-files/publish-availability-monitor.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: publish-availability-monitor
+  labels:
+    app: publish-availability-monitor
+    visualize: "true"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: publish-availability-monitor
+  template:
+    metadata:
+      labels:
+        app: publish-availability-monitor
+        version: "latest"
+        visualize: "true"
+    spec:
+      containers:
+      - name: publish-availability-monitor
+        image: coco/publish-availability-monitor:v0.24.17-k8s-version-rc7
+        imagePullPolicy: Always
+        resources:
+          limits:
+            memory: 512Mi
+        env:
+        - name: KAFKA_TOPIC
+          value: "PreNativeCmsPublicationEvents"
+        - name: KAFKA_PROXY_HOST
+          valueFrom:
+            configMapKeyRef:
+              name: global-config
+              key: kafka.proxy.url
+        - name: QUEUE_ADDR
+          value: "http://kafka:8082"
```

Comment:
I removed this one from the global config but you can add it back so we have all kafka addresses in one place
`  kafka.proxy.url.with.protocol: http://kafka:8082`